src/scss/_footer.scss

```diff
@@ -64,6 +64,13 @@
 
   .desktop-show {
     display: block;
+    .hiring-badge{
+      font-size: 12px;
+      font-weight: 600;
+      color: white;
+      background-color: #FF659D;
```

Comment:
Don't we have a sass color variable for this?

Reply:
yes we do, changed and pushed it :)